src/ucs/vfs/sock/vfs_sock.c

```diff
@@ -40,17 +41,23 @@ void ucs_vfs_sock_get_address(struct sockaddr_un *un_addr)
 
 int ucs_vfs_sock_mkdir(const char *sock_path, ucs_log_level_t log_level)
 {
-    char sock_path_dir[PATH_MAX];
+    char *sock_path_dir = (char *)ucs_malloc(PATH_MAX, "sock_path_dir");
```

Comment:
1. In C there is no need to cast from `void *`(in particular from `ucs_malloc`). This can be applied to all modifications
2. In this function there is a memory leak in case `mkdir` returns an error.

Reply:
done

Reply:
Outdated, done in other PRs